src/stream/scene_stream.cpp

```diff
@@ -89,6 +91,19 @@ int cmzn_scene_write(cmzn_scene_id scene,
 				SceneJsonExport jsonExport(scene);
 				outputStrings.push_back(jsonExport.getExportString());
 			}
+            else if (streaminformation_scene->getIOFormat() == CMZN_STREAMINFORMATION_SCENE_IO_FORMAT_STL)
+            {
+                number_of_entries = 1;
+                cmzn_scenefilter_id scenefilter = streaminformation_scene->getScenefilter();
+                outputStrings.push_back(export_to_stl(scene, scenefilter));
+            }
+            else if (streaminformation_scene->getIOFormat() == CMZN_STREAMINFORMATION_SCENE_IO_FORMAT_WAVEFRONT)
+            {
+                cmzn_scenefilter_id scenefilter = streaminformation_scene->getScenefilter();
+                outputStrings = export_to_wavefront(scene, scenefilter, 1);
```

Comment:
It seems wavefront can return multiple strings. Hence I expect need to loop over them and call `outputStrings.push_back()` to add to other resource strings.

Reply:
There is no loop over resource strings and export to wavefront returns all resources.  So I don't see a need to do any looping.

Reply:
OK, looks like it doesn't mirror region stream where you can add additional outputs with different IO types.